assets/scss/business-profile.scss

```diff
@@ -9,6 +10,88 @@
     justify-content: center;
   }
 
+  .business-menu {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-bottom: 16px;
+    margin-top: 8px;
+
+    .business-menu-button {
+      min-width: 0;
+      padding: 8px;
+      border-radius: 1px;
+      @include mdc-button-container-fill-color(#f4f4f4);
+    }
+
+    .material-icons {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      margin: 0;
+      font-size: 24px;
+      color: rgba(0, 0, 0, 0.5);
+    }
+
+  }
+
+  .mdc-dialog {
+
+    .mdc-dialog__container {
+      width: 100%;
+      margin: 0 16px;
+
+      @media screen and (min-width: 960px) {
+        max-width: 960px;
+      }
+
+      .mdc-dialog__surface {
+        position: absolute;
```

Comment:
move positioning to the container - the container expects to be centred - let's override that.